orttraining/orttraining/training_api/training_session.cc

```diff
@@ -58,6 +71,16 @@ Status TrainingSession::CreateCheckpointState(CheckpointState& chkpt_state, bool
   return Status::OK();
 }
 
+Status TrainingSession::SetLearningRate(float learning_rate) noexcept {
+  ORT_RETURN_IF_ERROR(optimizer_->SetLearningRate(learning_rate));
+
+  return Status::OK();
+}
+
+Status TrainingSession::SchedulerStep() noexcept {
+  return scheduler_->Step();
```

Comment:
if RegisterScheduler is not required when user want to use customized lr scheduler, then scheduler_ will be null, right?

Reply:
That is right. I think it makes sense to check for nullptr here. Thanks.